src/Paramore.Brighter/RelationDatabaseOutbox.cs

```diff
@@ -46,13 +49,29 @@ public void Add(
             int outBoxTimeout = -1,
             IAmABoxTransactionProvider<DbTransaction>? transactionProvider = null)
         {
-            var parameters = InitAddDbParameters(message);
-            WriteToStore(transactionProvider, connection => InitAddDbCommand(connection, parameters), () =>
+            var dbAttributes = new Dictionary<string, string>()
             {
-                logger.LogWarning(
-                    "MsSqlOutbox: A duplicate Message with the MessageId {Id} was inserted into the Outbox, ignoring and continuing",
-                    message.Id);
-            });
+                {"db.operation.parameter.message.id", message.Id}
+            };
+            var span = Tracer?.CreateDbSpan(
+                new OutboxSpanInfo(dbSystem, databaseName, OutboxDbOperation.Add, outboxTableName),
```

Comment:
The dbAttributes need to be set on the OutboxSpanInfo. It doesn't look like they are.